### PR TITLE
Added Carling-2017-112e

### PR DESCRIPTION
**Pull request by chrzyki**
A couple of OR concepts in here that we don't have yet and that seem different enough that they can't be linked to either or (in my opinion). Also a couple of interesting concepts that could be added, e.g.:

* PEARL
* SNARE
* MONSOON SEASON (we have WET SEASON, though)
* HEAD-HUNTING

### CODE REVIEW

File: concepticondata/conceptlists/Carling-2017-112e.tsv [hunk 1]
```diff
@@ -1,113 +1,113 @@
...
+Carling-2017-112e-78	78	1276	AUTUMN	autumn	5078	Seasons & nature	
+Carling-2017-112e-79	79	1537	SPRINGTIME	spring	5079	Seasons & nature	
+Carling-2017-112e-80	80			iron / metal	5080	Metals & materials	
+Carling-2017-112e-81	81	759	SILVER	silver	5081	Metals & materials	
+Carling-2017-112e-82	82	1369	GOLD	gold	5082	Metals & materials	
+Carling-2017-112e-83	83			pearl	5083	Metals & materials	
+Carling-2017-112e-84	84			cowry shell	5084	Metals & materials	
+Carling-2017-112e-85	85	1803	WOOD	wood	5085	Metals & materials	
+Carling-2017-112e-86	86	857	STONE	stone	5086	Metals & materials	
+Carling-2017-112e-87	87			leather / hide	5087	Metals & materials
```
Comment (LinguList) [inline concepticondata/conceptlists/Carling-2017-112e.tsv]: Leather and hide can be linked to leather.

**Review by LinguList — COMMENTED** [review]
I think you are right: the or-concepts should be added, even my leather / hide comment is probably not good. Specifically iron / metal is an important double-concept used in many societies. I'd skip head-hunting, as it is difficult to describe, and add the flying squirrell, as this is easy to define ("Gleithörnchen").

File: concepticondata/conceptlists/Carling-2017-112e.tsv (hunk 1 above)
Comment (chrzyki) [reply concepticondata/conceptlists/Carling-2017-112e.tsv]: So leave unlinked for now and I'll revisit after we've decided on the concepts that should be added?

**Comment by LinguList** [comment]
You could do all in one flush: add concepts,  as in the issue you added, add the concept list, add the linkings, and add the relations. 

What you could also do is: keep track of those concepts to be added by placing the `*` concept in your list. So before you make your PR, you can resolve this.

**Comment by chrzyki** [comment]
Okay, I'll add the new concepts with `*` and will add the new concepts at a later stage, is that okay?

**Comment by LinguList** [comment]
For me, definitely. It is a real brain-breaker to fiddle with multiple lists, adding concepts, not forgetting this, etc., therefore my longer discussion about this in the other thread. Ideally, adding starred glosses without Concepticon_ID could be parsed with some command that adds a template for empty definitions, etc., plus the concepticon_ids, so one could do definitions and the like later quickly in a rush...

**Comment by LinguList** [comment]
But that requires some coding, I guess, but maybe this is really an idea for concepticon-dev.

**Comment by chrzyki** [comment]
> For me, definitely. It is a real brain-breaker to fiddle with multiple lists, adding concepts, not forgetting this, etc., therefore my longer discussion about this in the other thread. Ideally, adding starred glosses without Concepticon_ID could be parsed with some command that adds a template for empty definitions, etc., plus the concepticon_ids, so one could do definitions and the like later quickly in a rush...

I agree. :) Updated with the starred glosses.

**Comment by LinguList** [comment]
So feel free to merge for the time being.

**Pull request merged**